orderly_evm_connector/websocket/orderly_socket_manager.py

```diff
@@ -112,12 +112,13 @@ def read_data(self):
                 op_code, frame = self.ws.recv_data_frame(True)
                 try:
                     _message = json.loads(frame.data)
+                    if "event" in _message:
+                        if _message["event"] == "ping":
+                            self._handle_heartbeat()
```

Comment:
Cosmetic, and matter of taste even:
```suggestion
                    if "event" in _message and _message["event"] == "ping":
                        self._handle_heartbeat()
```

Reply:
yeah, it looks better with an and